src/com/estafet/learning/US008/TradeInvoice.java

```diff
@@ -0,0 +1,20 @@
+package com.estafet.learning.US008;
+
+import java.util.Objects;
+
+class TradeInvoice extends Invoice {
+    TradeInvoice td = new TradeInvoice();
```

Comment:
I think it make sense to use `equals` method on an object created elsewhere, i.e. you instantiate it somewhere and compare

```
TradeInvoice td = new TradeInvoice(args);
TradeInvoice td2 = new TradeInvoice(args);

System.out.println(td.equals(td2));

```

Reply:
changed.